mne_rsa/viz.py

```diff
@@ -67,3 +69,168 @@ def plot_dsms(dsms, names=None, items=None, n_rows=1, cmap='viridis',
+                                    unified=False)
+
+    for i, (ax, _) in enumerate(my_topo_plot):
+        dsms_i = dsms[i]
+        dsms_i = distance.squareform(dsms_i)
+        ax.imshow(dsms_i, cmap=cmap)
+
+    if show:
+        fig.show()
+
+    return fig
+
+
+def plot_dsms_topos(dsms, info, time_windows=None, layout=None, figs=None,
+                    axis_facecolor='w', axis_spinecolor='w',
+                    fig_facecolor='w', figsize=(6.4, 4.8), cmap='viridis',
+                    show=False):
+    """ Plot DSMs on 2D MEG topographies for multiple time windows
+
+    Parameters
+    ----------
+    dsms: ndarray | numpy.memmap, shape (n_sensors, n_epochs, n_dsm_datapoint)
+        DSMs of MEG recordings; one DSM for each sensor and time point.
+    info: mne.io.meas_info.Info
+        Info object that contains meta data of MEG recordings.
+    time_windows: list of [int, int] | None, optional
+        List of time windows for each of which
+            the average DSM is calculated and plotted.
+        If the default, time_windoww=None, the average of DSMs of
+            all the time points is plotted.
+        Start of time window is inclusive while end exclusive.
+    layout: mne.channels.layout.Layout, optional
+        Layout objects containing sensor layout info.
+        The default, layout=None, will figure out layout based on info.
+    figs: list of matplotlib.pyplot.Figure | None, optional
+        Figure objects on which DSMs on 2D MEG topography are plotted.
+    axis_facecolor: str, optional
+        Face color of the each DSM. Defaults to 'w', white.
+    axis_spinecolor: str, optional
+        Spine color of each DSM. Defaults to 'w', white.
+    fig_facecolor: str, optional
+        Face color of the entire topography. Defaults to 'w', white.
+    figsize: tuple of float, optional
+        Figure size. The first element specify width and the second height.
+        Defaults to (6.4, 4.8).
+    cmap: str, optional
+        Colormap used for plotting DSMs. Defaults to 'viridis'.
+        Check matplotlib.pyplot.imshow for details.
+    show: bool, optional
+        Whether to display the generated figure. Defaults to False.
+
+    Returns
+    -------
+    figs: list of matplotlib.pyplot.Figure
+        Figure objects in each of which DSMs is plotted on 2D MEG topography
+            for each time window.
+    """
+    if len(dsms.shape) != 3:
+        raise ValueError("dsms have to be 3-dimensional ndarray or "
+                         "numpy.memmap, "
+                         "[n_sensors, n_epochs, n_dsm_datapoints]")
+    elif time_windows is None:
+        time_windows = [[0, dsms.shape[1]]]
+    elif not isinstance(time_windows, list):
+        raise TypeError("time has to be list of [int, int].")
+    elif not all(isinstance(i, list) for i in time_windows):
+        raise TypeError("time has to be list of [int, int].")
+    elif figs is None:
+        figs = [None]*len(time_windows)
```

Comment:
```suggestion
        figs = [] * len(time_windows)
```

Reply:
`[] * len(time_windows)` results in empty result `[]` instead of a list of `None`. As the rest of the code indexes on `figs`, I prefer not to change this. But if you want to avoid having a list consisting of `None`, I can modify other parts so that the code can work with `[] * len(time_windows) `.

Reply:
You're right. Ignore my comment.